notebooks/qg/passageqg_inference.ipynb

```diff
@@ -44,15 +44,14 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "2022-07-06 11:02:20 INFO: Downloading default packages for language: multilingual (multilingual)...\n",
-      "2022-07-06 11:02:20 INFO: File exists: /u/saneem/stanza_resources/multilingual/default.zip\n",
```

Comment:
Do we want these filenames with the full paths? Doesn't matter too much but just in case..